index.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright © 2020 Atomist, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// lib
+export {} from "./lib/bundle";
+export { buttonForCommand, menuForCommand } from "./lib/button";
+export {
+    execPromise,
+    spawnPromise,
+    killProcess,
+    SpawnPromiseOptions,
+    SpawnPromiseReturns,
+    WritableLog,
+} from "./lib/child_process";
+export {} from "./lib/context";
+export {} from "./lib/function";
+export { GraphQLClient, QueryOrLocation, Location } from "./lib/graphql";
+export {
+    Contextual,
+    CommandContext,
+    CommandHandler,
+    Configuration,
+    EventContext,
+    EventHandler,
+    HandlerStatus,
+} from "./lib/handler";
+export { HttpClient } from "./lib/http";
+export { debug, error, info, warn } from "./lib/log";
+export {
+    CommandMessageClient,
+    MessageClient,
+    RequiredMessageOptions,
+    MessageOptions,
+    Destinations,
+    SlackFileMessage,
+} from "./lib/message";
+export {
+    slackErrorMessage,
+    slackFooter,
+    slackInfoMessage,
+    slackSeparator,
+    slackSupportLink,
+    slackSuccessMessage,
+    slackTs,
+    slackQuestionMessage,
+    slackWarningMessage,
+} from "./lib/messages";
```

Comment:
Could/should these all be available under a single `slack` or `slackMessage` export?